src/main/java/net/glowstone/block/blocktype/BlockTnt.java

```diff
@@ -19,7 +20,7 @@
      * @param tntBlock The block to ignite.
      * @param ignitedByExplosion True if another explosion caused this ignition.
      */
-    public static void igniteBlock(GlowBlock tntBlock, boolean ignitedByExplosion) {
+    public static void igniteBlock(Block tntBlock, boolean ignitedByExplosion) {
```

Comment:
I think this is due to a merge mistake -- is there a reason for this change?

Reply:
The block doesn't actually have to be a GlowBlock for the method body to work, and widening the types will help simplify calling code and maximize plugin compatibility.